res/values/strings.xml

```diff
@@ -347,13 +362,26 @@
     <!-- mailing lists -->
     <string name="mailing_list">Mailing list</string>
     <string name="ask_show_mailing_list">Show mailing list \"%1$s\"?</string>
-
+    <string name="mailing_list_profile_info">Changes on mailing list name and image apply to this device only.</string>
 
     <!-- map -->
     <string name="filter_map_on_time">Show locations in time frame</string>
     <string name="show_location_traces">Show traces</string>
     <string name="add_poi">Send point of interest</string>
 
+    <!-- chat audit log -->
+    <string name="chat_audit_log_empty_message">System messages of this chat would appear here</string>
+    <string name="chat_audit_log_title">Audit Log for %1$s</string>
```

Comment:
```suggestion
    <string name="chat_audit_log_title">Chat Audit Log for %1$s</string>
```

just "Audit Log for Bjoern" looks scary to me :) also that thing is called "Chat Audit Log" in the menu, so, let's use the same in the title.